_posts/newsletters/2025-11-01-newsletter.md

```diff
@@ -0,0 +1,413 @@
+As with at GOOD25, the conference will kick off on Monday afternoon (March 9) with the optional Contributor Jam, an intensive deep dive into our code and documentation which will prepare attendees to contribute back to the Open OnDemand open source project."
+    links="Registration and the Call for Proposals are now open! We encourage an array of presentation modalities, including posters, prerecorded talks, and more (see website for details). Submissions are due on Dec 12, 2025.
+More information is available at [https://good2026.org](https://good2026.org)"
+%}
+
+<!-- end event boxes-->
+
+-----------------
+
+<a name="reads"></a>
+# 📚 **8. Featured Reads, Videos, and Podcasts**
+
+### 📑 Recent Publications
+
+- **Di Cosmo, R., Granger, S., Hinsen, K., Jullien, N., Le Berre, D., Louvet, V., Maumet, C., Maurice, C., Monat, R., & Rougier, N. P.** (2025). _"Stop treating code like an afterthought: Record, share and value it."_ Nature, 646(8084), 284–286. [Read the article.](https://doi.org/10.1038/d41586-025-03196-0)
+
+- **Academic Data Science Alliance** (2025). _"Finding Computing and Data Infrastructure to Get Started in Data-Intensive Research"_. [Read the post.](https://academicdatascience.org/resources/community-blog/get-started-in-data-intensive-research/)
+
+- **Kaiser, J., & Datta, S.** (2021). _"Probabilistic computing with p-bits."_ Applied Physics Letters, 119(15), 150503. [Read the article.](https://doi.org/10.1063/5.0067927)
+
+- **McMahon, P. L.** (2023). _"The physics of optical computing."_ Nature Reviews Physics, 5(12), 717–734. [Read the article.](https://doi.org/10.1038/s42254-023-00645-5)
+
+- **Stanford News** (2025), _"MingKwai prototype, the ‘origin of Chinese computing,’ finds a home at Stanford"_. [Read the post.](https://news.stanford.edu/stories/2025/05/mingkwai-chinese-typewriter-prototype-stanford-libraries)
+
+- **Nature Reviews Physics** (2025), _"Physics needs research software engineers."_ Nature Reviews Physics, 7(7), 349–349. [Read the editorial.](https://doi.org/10.1038/s42254-025-00852-2)
+
+- **Ramanathan, S., & Sengupta, A.** (2025). _"Introduction to Neuromorphic Computing."_ Cambridge University Press. [Pre-order the book.](https://www.cambridge.org/core/books/introduction-to-neuromorphic-computing/853942021D89F82AED182F9021E7E0F5)
+
+- **Saul, B.** (2025). _"Great expectations: Unifying Statistical Theory and Programming."_ arXiv. [Read the pre-print.](https://doi.org/10.48550/arXiv.2510.09853)
+
+- **Nature Portfolio** (2025). _"Six factors affecting reproducibility in life science research and how to handle them."_ [Read the editorial.](https://www.nature.com/articles/d42473-019-00004-y)
```

Comment:
I don't think this should be in the newsletter. It's not a peer-reviewed article or even an editorial, but an ad from a company.

Reply:
Now I'm confused because I wrote all of these by hand following the last mishap and now I can't find the original message that proposed it